README.md

```diff
@@ -1,54 +1,80 @@
+For the first iteration of this tool, I have decided to pursue the first idea and create a tool that will do the following:
 
-- [x] Define an MVP that will provide immediate value and a running backlog
+- Connect with Conjur / DAP containers
 
-- [x] Create the design and map out call chains and possible user interactions
+- Extract ENV variables
 
-For the first iteration of this tool, I have decided to pursue the first idea and create a tool that will do the following:
+    - *NOTE:* By default, the tool also supports DAP b/c from the functionality currently offered, the user can input 
+    the container name/id and receive the necessary logs and ENV variables which is environment agnostic
+    
+    - *NOTE:* For OSS, we are limited to setting the log levels as container ENV variables before OSS spin up
+
+- Parse and extract Conjur / DAP logs based on the type user requests
 
-- Connect with Conjur OSS containers and extract ENV variables
+- Return the user's relevant data in Conjur (if the user exists in Conjur, the Conjur policy where the user is defined, 
+their privileges, etc).
 
-- Parse and extract Conjur logs based on the type user requests
+- Return a result page with data
 
-- A .txt file will be returned to the user with the ENV variables listed at the top and all associated log entries
+## Phase 0
+
+- [x] Reach out to developers, PO/Ms, and SEs to understand the pain points and use this feedback to develop tooling that will feature in the project
+
+- [x] Define an MVP that will provide immediate value and a running backlog
+
+- [x] Create the design
 
 ## Design
 
+Class diagram:
 
-The following defines the sequence diagram for this project.   
+![Class Diagram](./classDiagram.png)
 
-![System Sequence Diagram](./troubleshootSystemSequenceDiagram.png)
+Sequence diagram:
 
-The following represnts the use case diagram for this project.
+![System Sequence Diagram](./troubleshootSystemSequenceDiagram.png)
 
-![Use Case Service Diagram](./useCaseServiceMapping.png)
+Activity diagram:
 
-### Open questions:
+![Activity Diagram](./activityDiagram.png)
 
-- I provided the Use Case Diagram above but do not feel that those (`Get Container Data` and `Extract Logs/Env`) are use cases as these items depend on Troubleshooting to operate correctly. Are there more use cases besides troubleshooting? 
+UI Mocks can be found here: https://www.figma.com/file/QcAxsC58Lh969NLunI8sTT/Authn2Mock?node-id=0%3A1
 
-- In the Troubleshooting class should I use a factory method that chooses between 2 different classes (DAP/Conjur) that implements the same interface (ConjurEnv)?
+- App will save state so that the user does not have to redefine query with each session
+ 
+### Open questions:
 
-- What language to write this project in?
\ No newline at end of file
+- What language to write this project in? 
+    
+    - Angular/Typescript, Node Docker API: https://www.npmjs.com/package/node-docker-api
```

Comment:
You plan this to be UI only project?

Reply:
no, just added a detail description of the backend